README.md

```diff
@@ -169,134 +137,116 @@ Some changes do not require an RFC:
+  and your team. Build consensus and integrate feedback. RFCs that have broad support 
+  are much more likely to make progress than those that don't receive any comments.
+8. Eventually, the team will decide whether the RFC is a candidate for inclusion
   in CDK/JSII.
-- RFCs that are candidates for inclusion in CDK/JSII will enter a "final comment
-  period" lasting 3 calendar days. The beginning of this period will be signaled
+9. RFCs that are candidates for inclusion in CDK/JSII will enter a "**final comment
+  period**" lasting 3 calendar days. The beginning of this period will be signaled
   by a team member adding a comment and label on the RFCs pull request.
-- An RFC can be modified based upon feedback from the team and community.
+10. An RFC can be modified based upon feedback from the team and community.
   Significant modifications may trigger a new final comment period.
-- An RFC may be rejected by the team after public discussion has settled and
+11. An RFC may be **rejected** by the team after public discussion has settled and
   comments have been made summarizing the rationale for rejection. A member of
-  the team should then close the RFCs associated pull request.
-- An RFC may be accepted at the close of its final comment period. A team member
+  the team will then close the PR and issue.
+12. An RFC may be accepted at the close of its final comment period. A team member
   will merge the RFCs associated pull request, at which point the RFC will
-  become 'ready'.
-
-If the submitter is someone from our CDK community (i.e., not core team member),
-a core team member will be assigned to 'champion' each proposal. They will
+  become 'approved'.
+13. At some point, someone will pick up the RFC for implementation. For major features
+  this usually requires devising a detailed implementation plan. To that end, submit
+  an **additional PR** on the RFC doc that fills in the "Implementation Plan" section.
+14. Once this PR is approved, the RFC will move to the 'implementing' state.
+    Usually we track implementation using GitHub projects.
+15. Once implementation is complete, the RFC moves to 'done', and it's issue is closed.
+
+> If the submitter is someone from our CDK community (i.e., not core team member),
+a core team member will be assigned to 'shephard' each proposal. They will
 generally be the ones updating the RFCs state in the tracking issue as it moves
 through the process. They can decide when a final comment period is triggered.
-
-On the other hand, if the submitter is a core team member, they will identify
+>
+> On the other hand, if the submitter is a core team member, they will identify
 another core team member, with consent, as their 'champion'. The champion would
 be the first contact for brainstorming, process and reviews. The core team
 would defer to the champion to do the first few rounds of reviews, after which
 the rest of the team should be engaged.
```

Comment:
Does the core team member decide on final comment period in this case, or the champion?

Reply:
The champion is the core team members. Let me rephrase.